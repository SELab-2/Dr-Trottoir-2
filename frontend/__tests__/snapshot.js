// __tests__/snapshot.js
//TODO: Just an example, to be deleted

import { render } from '@testing-library/react'
import Home from '../src/pages/index'

it('renders homepage unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
})
