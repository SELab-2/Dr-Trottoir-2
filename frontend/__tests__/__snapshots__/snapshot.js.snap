// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="main"
  >
    <h1>
      Ren√©.
    </h1>
    <div
      class="description"
    >
      Rene was hier.
    </div>
    <div
      style="height: 50%;"
    />
    <div
      class="description"
    >
      En ook hier.
    </div>
  </main>
</div>
`;
